tests/testAutopas/tests/selectors/tuningStrategy/PredictiveTuningTest.cpp

```diff
@@ -449,53 +136,36 @@ TEST_F(PredictiveTuningTest, testTuningThreeIterations) {
 /*
  * Tests the selection of the right optimumSearchSpace in the regard of the number of iterations without a test and
  * tuning. Two different configurations:
- *      - C08 is constant out of the optimum range (20).
- *      - Sliced is constant the optimum (10).
+ *      - first is constant out of the optimum range (20).
+ *      - second is constant the optimum (10).
  * In iteration three to six only sliced should be in _optimalSearchSpace.
  * In the seventh iteration lc_c08 and sliced should be in _optimalSearchSpace.
  */
-TEST_F(PredictiveTuningTest, testTooLongNotTested) {
-  unsigned int iteration = 0;
+TEST_F(PredictiveTuningTest, testLinearPredictionTooLongNotTested) {
```

Comment:
This does not work, because `tuneForSomeIterationsAndCheckAllTuned` always checks if `allConfigs` got selected in a tuningPhase, but in this test, only two configurations get used and not the three in `allConfigs`.

Reply:
added a possibility to specify "allConfigurations" to `tuneForSomeIterationsAndCheckAllTuned()`